src/test/field-element-creation-strategy.test.ts

```diff
@@ -0,0 +1,43 @@
+const {FieldElementCreationStrategy} = require('../form-element-creator/field-element-creation-strategy')
+
+test('create input element with attributes', () => {
+    const propertiesConfig = {
+        fields: [{
+            class: 'fields',
+            name: 'Username',
+            type: 'text',
+            placeholder: 'Username'
+        }]
+    };
+
+    const inputElement = new FieldElementCreationStrategy().createInput(propertiesConfig.fields[0]);
+
+    const expectedHTML = '<input type="text" name="Username" placeholder="Username">';
```

Comment:
wyzej w sekcje given

Reply:
wszystko co piszesz z palca czyli np. stala ktora jest wynikiem wywolania okreslonej funkcji z przekazanymi przez Ciebie parametrami ma byc w given.
Wszystko to co Ty dostarczasz recznie do fukkcji lub porownania jej wyniku ma byc w given.